doc/proposals/images-subspec.md

```diff
@@ -0,0 +1,190 @@
+---
```

Comment:
I'd suggest/recommend improve the first comment to open this PR in order to make the goal more clear. Following some ideas. 

> **Description of the change:**
> Proposal for new operator-sdk future to allow users to inform images and the common secondary resources required for it be created as all logic which would be required to follow up the best practices to manage them.
>
> **Motivation for the change:**
> Many operators as for example x and x will at the end have CRD's which will represent a deployment/pod of 
> an image:tag. So, I am as oper user I'd like to have a feature to make this process easier.

Reply:
The above is just an idea. But the goal would be in a simple way make clear what is the problem that you are trying to solve with.

Reply:
Sure, I'll go ahead and do this.